hook.sh

```diff
@@ -50,10 +49,20 @@ function deploy_challenge {
     existingRrdata=${existingRrdata//$'"""'/''}
 
     if [ "$existingName" == "_acme-challenge.$DOMAIN." ]; then
-        gcloud dns record-sets transaction remove --name $existingName --type TXT --ttl $existingTtl --zone $zonename -- "$existingRrdata" 
+        # Remove old txt token
+        gcloud dns record-sets transaction start --zone $zonename
+        gcloud dns record-sets transaction remove --name $existingName --type TXT --ttl $existingTtl --zone $zonename -- "$existingRrdata"
+        gcloud dns record-sets transaction execute --zone $zonename --format='value(id)'
```

Comment:
Is it necessary to perform a `transaction execute` here and a `transaction start` in line 59? Or can we also handle that all in one transaction?